Public/Copy-ADOUSecurity.ps1

```diff
@@ -0,0 +1,87 @@
+<#
```

Comment:
I believe synopsis should be within function above cmdletbinging().

Reply:
Corrected this and added credential param.